vcluster/deploy/topologies/air-gapped.mdx

```diff
@@ -16,241 +16,237 @@ import FipsConfig from '../../_partials/deploy/fips-config.mdx';
 import LicenseKey from '../../_partials/deploy/license_key.mdx';
 import ProAdmonition from '../../_partials/admonitions/pro-admonition.mdx';
 
-# vCluster Pro air-gapped install
+
+# Deploy vCluster in an air-gapped environment
 
 <ProAdmonition/>
 
-This document details the prerequisites and steps to install vCluster Pro into
-an `air-gapped` Kubernetes cluster without the platform agent running on the host
-cluster.
+This document details the steps to deploy vCluster into
+a Kubernetes cluster that doesn't have access to the internet. 
 
 ## Prerequisites
 
-<BasePrerequisites />
-  - `docker` (check with `docker version`)
-  - An offline license key for vCluster Pro (provided by Loft)
+- OCI-compliant private registry - A private registry accessible to both the Kubernetes host cluster and a separate, internet-connected machine.
+- An offline license key for vCluster - A LoftLabs license key is required to enable use with an air-gapped cluster.
```

Comment:
Only if they are using enterprise features. They could still do an air-gapped install of OSS following these instructions sans offline license key.